_posts/2025-11-11-cockpit-351.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Cockpit 351
+author: mvo
+date: '2025-11-11'
+tags: cockpit, machines
+slug: cockpit-351
+category: release
+summary: 'deleting firewall ports, restarting VMs, forwarding ports'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 351, and cockpit-machines 344:
+
+
+## Firewall: Additional ports can now be deleted
```

Comment:
Do we typically include `Page: ` in the headings? I thought we just do that for other packages